security/certs.go

```diff
@@ -129,7 +142,7 @@ func RunCreateCACert(certsDir string, keySize int) error {
 // RunCreateNodeCert is the entry-point from the command-line interface
 // to generate node certs and keys:
 // - node.server.{crt,key}: server cert with list of dns/ip addresses
-// - node.{crt,key}: client cert with "node" as the Common Name.
+// - node.client.{crt,key}: client cert with "node" as the Common Name.
```

Comment:
Cool. `(*nodeServer).executeCmd` would be where I would require this information. But even then, it would be great to somehow have access to the source address (after all, a client can have multiple active instances), which I'm not sure we can get without hacks.